utils/utils.go

```diff
@@ -283,3 +294,16 @@ func WrapIntoJsonRpcResponse(result interface{}) ([]byte, error) {
 
 	return json, nil
 }
+
+func WrapIntoJsonRpcErrorResponse(errorMessage string, data any) any {
```

Comment:
Error response is returned as an ordered JSON. Could you use a struct or something similar to keep the order?